frontend/src/router/index.js

```diff
@@ -190,40 +190,45 @@ router.beforeEach((to, _from, next) => {
 
   function validateAndExecute(nextRouteInError, to) {
```

Comment:
I'm not sure this method is required as the first parameter isn't even used. We could just inline the code to run if (to.meta.requiresAuth)

Reply:
Confirming completed.